honeybee_energy/hvac/heatcool/_base.py

```diff
@@ -52,6 +54,11 @@ class _HeatCoolBase(_TemplateSystem):
             'DHWBaseboard', 'GasHeaters', 'ResidentialHPNoCool', 'ResidentialFurnace'
         )
 
+    def __init__(self, identifier, vintage='ASHRAE_2019', equipment_type=None):
+        # initialize base HVAC system properties
+        super(_HeatCoolBase, self).__init__(identifier, vintage, equipment_type)
```

Comment:
I had a question on that actually? I noticed that pattern being used throughout the HB code, and that seems to work for single inheritance fine. Are you trying to enforce single inheritance by using that syntax? I don't have/plan to have any multiple inheritance, so is fine by me either way. Just wondering?

The [Python docs](https://docs.python.org/2.7/library/functions.html?highlight=super#super) seem to suggest that using `super(*child-class-name*, self).__init__()` is the preferred syntax? But I don't think I really see why that would be? They do say that:

> "...super can be used to refer to parent classes without naming them explicitly, thus making the code more maintainable."
 
which, I'm not sure I follow why that is true, at least in simple cases like the one here?

Regardless: For consistency with the rest of HB and since I don't think we want/need multiple inheritance: Yes I can just change all my super() calls to just the class instead. That's just fine by me.

Reply:
Gotcha - for consistency I'll rewrite this so that this matches your existing pattern with the Parent being called directly.